packages/block-library/src/post-featured-image/edit.js

```diff
@@ -228,38 +238,20 @@ export default function PostFeaturedImageEdit( {
 					media={ media }
 				/>
 			</InspectorControls>
-			<InspectorControls>
-				<ToolsPanel
-					label={ __( 'Settings' ) }
-					resetAll={ () => {
-						setAttributes( {
-							isLink: false,
-							linkTarget: '_self',
-							rel: '',
-						} );
-					} }
-					dropdownMenuProps={ dropdownMenuProps }
-				>
-					<ToolsPanelItem
-						label={
-							postType?.labels.singular_name
-								? sprintf(
-										// translators: %s: Name of the post type e.g: "post".
-										__( 'Link to %s' ),
-										postType.labels.singular_name
-								  )
-								: __( 'Link to post' )
-						}
-						isShownByDefault
-						hasValue={ () => !! isLink }
-						onDeselect={ () =>
+			{ ( featuredImage || isDescendentOfQueryLoop ) && (
```

Comment:
```suggestion
			{ ( featuredImage || isDescendentOfQueryLoop || ! postId ) && (
```

I think this conditional statement is not enough. The settings panel needs to be in case of templates (e.g. single template), i.e. without post ID, otherwise the link won't be displayed in the single template for example.

![image](https://github.com/user-attachments/assets/eccc7d34-e781-46d3-b8a8-fe6b6c4fb109)